docs/concepts/aggregation.md

```diff
@@ -0,0 +1,165 @@
+able to use `kubectl` to fetch resources from newly registered APIs.
+
+Controller managers also commonly use discovery information to determine
+whether or not they should run: if the resources that they require are not
+present of the cluster, controller managers can just fail to start.
+
+You can see the list of API group versions returned by discover using the
+`kubectl api-versions` command.
+
+## Registering APIs
+
+In order for your API to appear in the discovery information served by the
+aggregator, it must be registered with the aggregator.  In order to do
+this, the aggregator exposes a normal Kubernetes API group called
+`apiregistration.k8s.io`, with a single resource, APIService.
+
+Each APIService corresponds to a single group-version, and different
+versions of a single API can be backed by different APIService objects.
+
+Let's take a look at the APIService for `wardle/v1alpha1`, using `kubectl
+get apiservice v1alpha1.wardle -o yaml`:
+
+```yaml
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.wardle
+  ...
+spec:
+  caBundle: <base64-encoded-serving-ca-certificate>
+  group: wardle
+  version: v1alpha1
+  priority: 100
+  service:
+    name: wardle-server
+    namespace: wardle-namespace
+status:
+  ...
+```
+
+Notice that this is a Kubernetes API object like any other: it has a name,
+a spec, a status, etc.  There are several important fields in the spec.
+
+The first important field is `caBundle`.  This is the base64-encoding
+version of a CA certificate that can be used to verify the serving
+certificates of the API server.  The aggregator will check that the
+serving certificates are for a hostname of `<service>.<namespace>.svc` (in
+the case of the APIService above, that's
+`wardle-server.wardle-namespace.svc`).
+
+Next are the `group` and `version` fields.  These determine which
+group-version the APIService describes.
+
+The `priority` field indicates how the aggregator sorts and prioritizes
+discovery information.
+
+Finally, the `service` field determines how the aggregator actually
+connects to the addon API server.  It will look up the service IP of the
+service described, and connect to that.  As mentioned above, however, it
+validates certificates based on hostname, so your certificates do not
+necessarily need to include the IP address of the service.
+
+## Proxying
+
+In addition to serving discovery information and registering API groups
+and servers, the aggregator acts as proxy.
+
+The aggregator "natively" serves discovery information that lets us list
+the available API groups, and the corresponding versions.  For other
+requests, such as fetching the available resources in a group-version, or
+making a request against an API, the aggregator contacts a registered API
+server.
+
+When the aggregator receives a request that it needs to proxy, it first
+performs authentication (but not authorization) using the authentication
```

Comment:
it's behind the full chain. It does a first pass at authorization.  This does *not* mean your addon API server can skip authorization. Remember, you're exposing a service, anyone can hit it.